index.js

```diff
@@ -1 +1,13 @@
-export * from './dist/auro-combobox';
+import { AuroCombobox } from './src/auro-combobox.js';
+
+/**
+ * Register Custom Element.
+ * @param {Object} name - Name to use for custom element.
+ * @returns {void}
+ */
+export function registerComponent(name) {
+  // alias definition
+  if (!customElements.get(name)) {
+    customElements.define(name, class extends AuroCombobox {});
+  }
+}
```

Comment:
You need to update rollup config, see file

- https://github.com/AlaskaAirlines/auro-hyperlink/blob/main/rollup.config.mjs

Also be sure to update the .npmignore to include the proper resource from the ./src dir.